bin/MadGraph5_aMCatNLO/cards/production/13TeV/HZaTo2l2g/HZaTo2l2g_proc_card.dat

```diff
@@ -0,0 +1,10 @@
+import model ALP
+
```

Comment:
The model is 4 FS, no? Below you treat is as 5FS. You might want to add a -no-b-mass restriction card to the ufo.

Reply:
From here we can get the model tar file: 
https://cms-project-generators.web.cern.ch/cms-project-generators/
We checked the particles.py file and the b-quark is defined. So I think the model should be 5FS.

Reply:
The b-quark should be included in the proton definition. And from what I see, it's not.
You should either redefine p with b b~ or switch 5fs settings to 4fs

<img width="950" alt="image" src="https://user-images.githubusercontent.com/17442656/123365265-e94c8a80-d5b0-11eb-8560-c73eb5ea2b2f.png">

Reply:
Yeah, in our proc_card, we redefine p withe b b~
![image](https://user-images.githubusercontent.com/41952890/123365537-05e4c480-d5a9-11eb-9b05-6b272a70be24.png)

Reply:
Ah sorry. I think I misunderstood Alexander's comments.
So for 5FS, it is assumed that u d c s b are effectively massless (4FS u d c s) following the definitions of PDF.

But in your UFO model, all masses are treated to have mass, and most disturbingly bottom mass is quite big. I think that's what @agrohsje is pointing out.

@Alexander : I somehow got misconception that if you define "p = p b b~" in the proc card, MadGraph will automatically reset the bottom masses to 0 but it seems not.

Reply:
So what's your suggestion? Should we change to 4FS?

Reply:
@zebingwang the b is not part of the p in this case so it should be 4FS. you can do 
import model ALP
generate p p > t t~ 
do you see 
b b > t t~. 
I don't think so. So the default model should be 4FS and you need to actively make it 5FS if that is your preferred choice.